Description: Allow SYSTEMML_INSTALL_PATH to be empty
 Simply prevent brahms from showing an error if this path is empty. This is
 Debian package specific.
Author: Seb James <seb.james@sheffield.ac.uk>
Forwarded: No need to forward.
Last-Update: 2014-02-22

--- a/framework/engine/support/environment.cpp
+++ b/framework/engine/support/environment.cpp
@@ -312,7 +312,7 @@
 
 			//	construct list of namespace roots
 			string NamespaceRoots = brahms::os::getenv("SYSTEMML_INSTALL_PATH");
-			if (!NamespaceRoots.length()) ferr << E_INSTALLATION << "env var SYSTEMML_INSTALL_PATH not set";
+			//if (!NamespaceRoots.length()) ferr << E_INSTALLATION << "env var SYSTEMML_INSTALL_PATH not set";
 			NamespaceRoots += brahms::os::PATH_SEPARATOR;
 			NamespaceRoots += "Namespace";
 			string s = gets("NamespaceRoots");
